yt/frontends/artio/definitions.py

```diff
@@ -27,7 +33,7 @@
     "stars": "STAR",
     "nbody": "N-BODY",
 }
-art_to_yt = dict(zip(yt_to_art.values(), yt_to_art.keys()))
+art_to_yt = dict(zip(yt_to_art.values(), yt_to_art.keys(), strict=True))
```

Comment:
This is correct behavior -- but it makes me realize that we are using a dict for a 1:1 mapping, which is not necessarily true.  Fortunately it is here!